backend/test.py

```diff
@@ -0,0 +1,4 @@
+import subprocess
+
+p = subprocess.Popen("../isisTest/isisTest")
+q = subprocess.Popen("../jv2/jv2")
```

Comment:
Not that it really matters, but perhaps `backend` and `frontend` rather than `p` and `q`?